packages/ui/src/components/va-stepper/VaStepper.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <div
+    class="va-stepper"
+    :class="{ 'va-stepper--vertical': $props.vertical }"
+  >
+    <ol
+      class="va-stepper__navigation"
+      :class="{ 'va-stepper__navigation--vertical': $props.vertical }"
+    >
+      <template
+        v-for="(step, i) in $props.steps"
+        :key="i"
+      >
+        <slot
+          v-if="i > 0"
+          name="divider"
+          v-bind="stepControls"
```

Comment:
Can we have the same bind for each slot?

Reply:
I'll make divider slots same as step slot, but for other slots it's either impossible or doesn't make sense. For example no reason to have is isCompleted and is isActive on step content because only active step's content appears on the screen.

Reply:
Actually nvm, I'll just do it anyway.